File: test/resources/codes/0000002/accepted/main.c
#include <math.h>
#include <stdio.h>
int main()
{
    int n, i, flag = 1;
 
    scanf("%d", &n);
 
    for (i = 2; i <= sqrt(n); i++) {

        if (n % i == 0) {
            flag = 0;
            break;
        }
    }
 
    if (n <= 1)
        flag = 0;
 
    if (flag == 1) {
        printf("TRUE");
    }
    else {
        printf("FALSE");
    }
 
    return 0;
}

File: test/resources/codes/0000002/accepted/main.py
from math import sqrt
# n is the number to be check whether it is prime or not
n = int(input())

prime_flag = 0
  
if(n > 1):
    for i in range(2, int(sqrt(n)) + 1):
        if (n % i == 0):
            prime_flag = 1
            break
    if (prime_flag == 0):
        print("TRUE")
    else:
        print("FALSE")
else:
    print("FALSE")